homeassistant/components/shelly/coordinator.py

```diff
@@ -616,7 +657,9 @@ def _async_handle_update(
         self, device_: RpcDevice, update_type: RpcUpdateType
     ) -> None:
         """Handle device update."""
-        if update_type is RpcUpdateType.INITIALIZED:
+        if update_type is RpcUpdateType.ONLINE:
```

Comment:
Looks like this line is not covered

Reply:
Its actually 663-664
codecov lies

homeassistant/components/shelly/coordinator.py            386      6    98%   411-412, 626, 650, 663-664